src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -1,14 +1,146 @@
 import _ from 'lodash'
 import { a, div, hh } from 'react-hyperscript-helpers'
-import { buttonPrimary, link } from 'src/components/common'
+import { buttonPrimary, contextMenu, link } from 'src/components/common'
 import { icon, spinner } from 'src/components/icons'
+import { NotebookCreator, NotebookDeleter, NotebookDuplicator } from 'src/components/notebook-utils'
+import ShowOnClick from 'src/components/ShowOnClick'
 import { DataTable } from 'src/components/table'
 import { Buckets, Leo } from 'src/libs/ajax'
 import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
-import { Component, Fragment } from 'src/libs/wrapped-components'
+import { Component, Fragment, Interactive } from 'src/libs/wrapped-components'
 
 
+const NotebookCard = hh(class NotebookCard extends Component {
+  render() {
+    const { namespace, name, updated, listView, notebookAccess, bucketName, clusterUrl, wsName, reloadList } = this.props
+    const { renamingNotebook, copyingNotebook, deletingNotebook } = this.state
+    const printName = name.slice(10, -6) // removes 'notebooks/' and the .ipynb suffix
+
+    const hideMenu = () => this.notebookMenu.setVisibility(false)
+
+    const notebookMenu = ShowOnClick({
+        ref: instance => this.notebookMenu = instance,
+        disabled: !notebookAccess,
```

Comment:
When I click on a disabled card, I get no indication of why it is disabled. Might be worth filing a bug and addressing later.